src/main/java/com/depromeet/domain/missionRecord/api/MissionRecordController.java

```diff
@@ -27,4 +31,12 @@ public ResponseEntity<Long> missionRecordCreate(
         Long missionRecordId = missionRecordService.createMissionRecord(request);
         return ResponseEntity.status(HttpStatus.CREATED).body(missionRecordId);
     }
+
+    @Operation(summary = "미션 기록 조회", description = "미션 기록을 조회합니다.")
+    @GetMapping
```

Comment:
미션 기록 조회에 기본 endpoint는 미션 단건이 더 맞는거같아서 아래와 같이 바꾸는건 어떨까요?
(미션 상세 조회도 스펙에 있기도하고)

```suggestion
    @Operation(summary = "미션 기록 캘린더 조회", description = "년, 월을 입력받아 미션 기록을 캘린더로 조회합니다")
    @GetMapping("/calender")
```

Reply:
훔.. 이건 재현님도 같이 의논을 해봐야 할 거 같은데
@uwoobeat 어떻게 생각하신가요?

Reply:
이거는 면대면으로 이야기하는 게 좋을 것 같아요
이따가 시간나면 게더에서 이야기 ㄱㄱ하시죠